apps/api-extractor/src/extractor/IExtractorConfig.ts

```diff
@@ -142,6 +142,57 @@ export interface IExtractorApiJsonFileConfig {
   outputFolder?: string;
 }
 
+/**
+ * Configures how the package typings (*.d.ts) will be generated.
+ * @remarks
+ * API Extractor can generate a single unified *.d.ts file that contains all
+ * the exported typings for the package entry point.  It can also remove
+ * \@alpha \@beta \@internal definitions depending on the release type.
+ *
+ * @beta
+ */
+export interface IExtractorPackageTypingsConfig {
+  /**
+   * Whether to generate package typings.  The default is false.
+   */
+  enabled: boolean;
+
+  /**
+   * Specifies where the *.d.ts files should be written.
+   *
+   * The default value is "./dist"
+   */
+  outputFolder?: string;
+
+  /**
+   * Specifies the output filename for an internal release.
+   *
+   * @remarks
+   * The default value is "index-internal.d.ts".  This file will contain all definitions that
+   * are reachable from the entry point.
+   */
+  internalFilename?: string;
```

Comment:
Make it more clear that this gets appended to `outputFolder`.

Reply:
You could consider renaming these to FilePath instead of Filename

---
In reply to: [152421826](https://github.com/Microsoft/web-build-tools/pull/448#discussion_r152421826) [](ancestors = 152421826)